### PR TITLE
Add readme and application models

### LINKED ISSUE #3: Application Models
Add models that will be established in [this issue](https://github.com/devinim-istanbul/shopping-list/issues/2)

### PR DESCRIPTION
**Pull request by yavrib**
Resolves #3

### CODE REVIEW

File: src/models/base/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class BaseModel {
+  constructor(id, name) {
+    this.id = id;
```
Comment (dincozdemir) [inline src/models/base/index.js]: Objects in **firebase** have the following structure:
```
houses: {
    id1: {
        users: {
            id1: { ... }
            id2: { ... }
        }
    },
    id2: { ... }
}
```
@erolkaftanoglu would we have a problem with this model?

**Comment by dincozdemir** [comment]
[Application Models](https://trello.com/c/UVRlXYHp/14-application-models)

File: src/models/base/index.js (hunk 1 above)
Comment (yavrib) [reply src/models/base/index.js]: Wait, I didn't actually get this, what is id1 and id2?

Comment (yavrib) [reply src/models/base/index.js]: I got it now, what is your suggestion?

Comment (dincozdemir) [reply src/models/base/index.js]: The objects themselves don't include the id values. Id values are inner object variables in the parent object. By this logic the object models should'nt be including an id field.

**Review by dincozdemir — APPROVED** [review]

**Pull request merged**